src/rust/ide/src/view/node_editor.rs

```diff
@@ -237,6 +244,7 @@ impl GraphEditorIntegratedWithController {
             let is_handling_notification = handle_notification.is_running;
             is_hold <- is_handling_notification.all_with(&invalidate.is_running, |l,r| *l || *r);
             _action <- editor_outs.node_removed             .map2(&is_hold,node_removed);
+            _action <- editor_outs.nodes_collapsed          .map2(&is_hold,nodes_collapsed);
             _action <- editor_outs.node_entered             .map2(&is_hold,node_entered);
             _action <- editor_outs.node_exited              .map2(&is_hold,node_exited);
             _action <- editor_outs.connection_added         .map2(&is_hold,connection_created);
```

Comment:
why there's such a lot of spaces between `nodes_...` and `.map2(...)`

Reply:
It's part of a larger alignment block:
```rust
            _action <- editor_outs.node_removed             .map2(&is_hold,node_removed);
            _action <- editor_outs.nodes_collapsed          .map2(&is_hold,nodes_collapsed);
            _action <- editor_outs.node_entered             .map2(&is_hold,node_entered);
            _action <- editor_outs.node_exited              .map2(&is_hold,node_exited);
            _action <- editor_outs.connection_added         .map2(&is_hold,connection_created);
            _action <- editor_outs.visualization_enabled    .map2(&is_hold,visualization_enabled);
            _action <- editor_outs.visualization_disabled   .map2(&is_hold,visualization_disabled);
            _action <- editor_outs.connection_removed       .map2(&is_hold,connection_removed);
            _action <- editor_outs.node_position_set_batched.map2(&is_hold,node_moved);
            _action <- editor_outs.node_expression_set      .map2(&is_hold,node_expression_set);

```